python/lance_namespace_urllib3_client/test/test_alter_transaction_action.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
```

Comment:
Do we really need these test files? Can we drop them?

Reply:
I ended up leaving those because it was able to at least check the generated files can be imported. The main issue of the conflict was due to the Makefile script bug